ironfish/src/workerPool/tasks/buildTransaction.ts

```diff
@@ -12,44 +12,44 @@ import { WorkerTask } from './workerTask'
 
 export class BuildTransactionRequest extends WorkerMessage {
   readonly transaction: RawTransaction
-  readonly proofGenerationKey: string
+  readonly proofAuthorizingKey: string
   readonly viewKey: string
   readonly outgoingViewKey: string
 
   constructor(
     transaction: RawTransaction,
-    proofGenerationKey: string,
+    proofAuthorizingKey: string,
     viewKey: string,
     outgoingViewKey: string,
     jobId?: number,
   ) {
     super(WorkerMessageType.BuildTransaction, jobId)
     this.transaction = transaction
-    this.proofGenerationKey = proofGenerationKey
+    this.proofAuthorizingKey = proofAuthorizingKey
     this.viewKey = viewKey
     this.outgoingViewKey = outgoingViewKey
   }
 
   serializePayload(bw: bufio.StaticWriter | bufio.BufferWriter): void {
-    bw.writeBytes(Buffer.from(this.proofGenerationKey, 'hex'))
+    bw.writeBytes(Buffer.from(this.proofAuthorizingKey, 'hex'))
     bw.writeBytes(Buffer.from(this.viewKey, 'hex'))
     bw.writeBytes(Buffer.from(this.outgoingViewKey, 'hex'))
     bw.writeBytes(RawTransactionSerde.serialize(this.transaction))
   }
 
   static deserializePayload(jobId: number, buffer: Buffer): BuildTransactionRequest {
     const reader = bufio.read(buffer, true)
-    const proofGenerationKey = reader.readBytes(VIEW_KEY_LENGTH).toString('hex')
+    const proofAuthorizingKey = reader.readBytes(ACCOUNT_KEY_LENGTH).toString('hex')
```

Comment:
nit: Doesn't need to be done here, but maybe I'm realizing we should somehow merge `KEY_LENGTH` and `ACCOUNT_KEY_LENGTH`